data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Парэм", "Parham" ],
+      [ "Патино", "Patino" ],
+      [ "Пауерс", "Powers" ],
+      [ "Пате", "Pate" ],
+      [ "Пейн", "Payne" ],
+      [ "Пейнтер", "Painter" ],
+      [ "Пенс", "Pence" ],
+      [ "Стамп", "Stump" ],
+      [ "Эш", "Ash" ],
+      [ "Пералес", "Perales" ],
+      [ "Кросс", "Cross" ],
+      [ "Пеппер", "Pepper" ],
+      [ "Перселл", "Purcell" ],
+      [ "Сэндс", "Sands" ],
+      [ "Пиментел", "Pimentel" ],
+      [ "Пиплс", "Peoples" ],
+      [ "Пирес", "Pires" ],
+      [ "Роуч", "Roach" ],
+      [ "Вейр", "Weir" ],
+      [ "Карпентер", "Carpenter" ],
+      [ "Уэзерс", "Weathers" ],
+      [ "Ринг", "Ring" ],
+      [ "Филд", "Field" ],
+      [ "Баттс", "Butts" ],
+      [ "Портер", "Porter" ],
+      [ "Пратт", "Pratt" ],
+      [ "Принс", "Prince" ],
+      [ "Париш", "Parish" ],
+      [ "Форман", "Foreman" ],
+      [ "Рид", "Read" ],
+      [ "Пруэтт", "Pruett" ],
+      [ "Берд", "Bird" ],
+      [ "Пуарье", "Poirier" ],
+      [ "Крэйвен", "Craven" ],
+      [ "Бланк", "Blank" ],
+      [ "Пэрротт", "Parrott" ],
+      [ "Пэтти", "Petty" ],
+      [ "Бутчер", "Butcher" ],
+      [ "Ракер", "Rucker" ],
+      [ "Шелл", "Shell" ],
+      [ "Ралстон", "Ralston" ],
+      [ "Эрли", "Early" ],
+      [ "Ранхель", "Rangel" ],
+      [ "Раст", "Rust" ],
+      [ "Комбс", "Combs" ],
+      [ "Ратледж", "Rutledge" ],
+      [ "Редмен", "Redman" ],
+      [ "Рентерия", "Renteria" ],
+      [ "Риверс", "Rivers" ],
+      [ "Роарк", "Roark" ],
+      [ "Робисон", "Robison" ],
+      [ "Хорн", "Horn" ],
+      [ "Родос", "Rhodes" ],
+      [ "Роквелл", "Rockwell" ],
+      [ "Ролз", "Rawls" ],
+      [ "Ропер", "Roper" ],
+      [ "Роуленд", "Rowland" ],
+      [ "Роша", "Rocha" ],
+      [ "Гроув", "Grove" ],
+      [ "Гровс", "Groves" ],
+      [ "Роэ", "Roe" ],
+      [ "Роял", "Royal" ],
+      [ "Рупп", "Rupp" ],
+      [ "Фиш", "Fish" ],
+      [ "Рэгланд", "Ragland" ],
+      [ "Рэгсдейл", "Ragsdale" ],
+      [ "Рэми", "Ramey" ],
+      [ "Рэнкин", "Rankin" ],
+      [ "Рэтлифф", "Ratliff" ],
+      [ "Саймонс", "Simons" ],
+      [ "Салдивар", "Saldivar" ],
+      [ "Сальмон", "Salmon" ],
+      [ "Самора", "Zamora" ],
+      [ "Самуэль", "Samuel" ],
+      [ "Бонд", "Bond" ],
+      [ "Селф", "Self" ],
+      [ "Севейдж", "Savage" ],
+      [ "Норт", "North" ],
+      [ "Сент-Джон", "St. John" ],
+      [ "Сильвер", "Silver" ],
+      [ "Сильвестр", "Sylvester" ],
+      [ "Сильвия", "Silvia" ],
+      [ "Стронг", "Strong" ],
+      [ "Синглетарий", "Singletary" ],
+      [ "Стейплс", "Staples" ],
+      [ "Слак", "Slack" ],
+      [ "Слаутер", "Slaughter" ],
+      [ "Слейтер", "Slater" ],
+      [ "Херд", "Heard" ],
+      [ "Смоллвуд", "Smallwood" ],
+      [ "Соарес", "Soares" ],
+      [ "Спейн", "Spain" ],
+      [ "Спраг", "Sprague" ],
+      [ "Старнес", "Starnes" ],
+      [ "Стейли", "Staley" ],
+      [ "Стейнберг", "Steinberg" ],
+      [ "Стейтон", "Staton" ],
```

Comment:
Apparently, this could be either [`Стэтон`](https://ru.wikipedia.org/wiki/%D0%A1%D1%82%D1%8D%D1%82%D0%BE%D0%BD,_%D0%90%D0%B0%D1%80%D0%BE%D0%BD) or [`Стэйтон`](https://ru.wikipedia.org/wiki/%D0%A1%D1%82%D1%8D%D0%B9%D1%82%D0%BE%D0%BD,_%D0%9A%D1%8D%D0%BD%D0%B4%D0%B8). The latter is what I would personally go for, given how the former sounds nothing like the original.